kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/document/datadictionary/PurchaseOrderAmendmentDocument.xml

```diff
@@ -1,10 +1,31 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<beans xmlns="http://www.springframework.org/schema/beans" 
-xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
-xmlns:p="http://www.springframework.org/schema/p" 
-xmlns:dd="http://rice.kuali.org/dd" 
-xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd         http://rice.kuali.org/dd         http://rice.kuali.org/dd/dd.xsd">
+<?xml version="1.0" encoding="UTF-8"?><beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xmlns:dd="http://rice.kuali.org/dd" xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd         http://rice.kuali.org/dd         http://rice.kuali.org/dd/dd.xsd">
 
+
+  <bean id="PurchaseOrderAmendmentDocument" parent="PurchaseOrderAmendmentDocument-parentBean">
+    <property name="documentClass" value="edu.arizona.kfs.module.purap.document.PurchaseOrderAmendmentDocument"/>
+    <property name="baseDocumentClass" value="edu.arizona.kfs.module.purap.document.PurchaseOrderAmendmentDocument"/>
+    <property name="attributes">
+      <list merge="true">
+        <ref bean="PurchaseOrderAmendmentDocument-routeCode"/>
+      </list>
+    </property>
+  </bean>
+    
+    
+    <!-- Attribute Definitions -->
+   <bean id="PurchaseOrderAmendmentDocument-routeCode" parent="RequisitionDocument-routeCode-parentBean"/>
+   
+   <bean id="RequisitionDocument-routeCode-parentBean" abstract="true" parent="AttributeDefinition"> 
```

Comment:
Should be:
bean id="PurchaseOrderAmendmentDocument-routeCode" parent="PurchaseOrderAmendmentDocument-routeCode-parentBean"
bean id="PurchaseOrderAmendmentDocument-routeCode-parentBean" abstract="true" parent="AttributeDefinition"